### PR TITLE
net-mail/isync: add 1.5.0, drop 1.4.4

### PR DESCRIPTION
**Pull request by akpall**
Added 1.5.0 ebuild (bump from 1.4.4-r1) and updated 9999
- EAPI 7 -> 8
- Changed all keywords to testing
- Removed missing blocker
- Changed BDEPEND to require ">=dev-lang/perl-5.14"
- Removed backported openssl3-eof.patch from ebuild

Dropped 1.4.4 due to it being superseded by 1.4.4-r1

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: net-mail/isync/isync-1.5.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+IUSE="berkdb sasl ssl zlib"
+
+RDEPEND="
+	berkdb? ( >=sys-libs/db-4.2:= )
+	sasl?	( dev-libs/cyrus-sasl )
+	ssl?	( >=dev-libs/openssl-0.9.6:0= )
+	zlib?	( sys-libs/zlib:0= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND=">=dev-lang/perl-5.14"
```
Comment (dlan17) [inline net-mail/isync/isync-1.5.0.ebuild:28]: specify version here isn't necessary (but doesn't hurt), as the minimal version in tree is 5.38

**Comment by dlan17** [comment]
generally, this ebuild looks good, and I've tested locally, it works for me..

**Comment by sergeyklay** [comment]
Hi @akhuettel,

I am reaching out to report a segmentation fault issue with the current version of isync available in Gentoo. After setting up a minimal configuration according to the man page, running mbsync results in a segmentation fault. Upon investigation, I found that this issue has been documented in both the Arch Linux and upstream communities:
1. A detailed description of the problem and proposed patches can be found in the isync-devel mailing list ([sourceforge.net](https://www.mail-archive.com/isync-devel@lists.sourceforge.net/msg04278.html)​).
2. This issue also affects Arch Linux users, as outlined in their packaging issue tracker ([archlinux.org](https://gitlab.archlinux.org/archlinux/packaging/packages/isync/-/issues/1)).

It seems that the problem arises due to a bug in the current release, which has already been resolved upstream with a patch. Given that Gentoo is currently using an older version, would it be possible to release a new version of the isync package to include these fixes?

Thank you for your time and consideration.

**Comment by dlan17** [comment]
> Hi @akhuettel,
> 
> I am reaching out to report a segmentation fault issue with the current version of isync available in Gentoo. After setting up a minimal configuration according to the man page, running mbsync results in a segmentation fault. Upon investigation, I found that this issue has been documented in both the Arch Linux and upstream communities:
> 
>     1. A detailed description of the problem and proposed patches can be found in the isync-devel mailing list ([sourceforge.net](https://www.mail-archive.com/isync-devel@lists.sourceforge.net/msg04278.html)​).
> 
>     2. This issue also affects Arch Linux users, as outlined in their packaging issue tracker ([archlinux.org](https://gitlab.archlinux.org/archlinux/packaging/packages/isync/-/issues/1)).
> 
> 
> It seems that the problem arises due to a bug in the current release, which has already been resolved upstream with a patch. Given that Gentoo is currently using an older version, would it be possible to release a new version of the isync package to include these fixes?
> 
> Thank you for your time and consideration.

I see this issue has been already fixed in 1.5.0, so should be solved with this PR merged
haven't back ported this patch to 1.4.4, but let me know if it's necessary..

**Comment by sergeyklay** [comment]
Thank you! It work like a charm :)

**Pull request closed without merging**